package proxy;
option cc_generic_services = true;

message proxy_message {
    optional string client_id = 1; // use just name of connection...for example
    optional bytes  data      = 2;
}

message empty { }

service transmitter {
    rpc send_to(proxy_message) returns (proxy_message);
    rpc reg_for_proxy(empty) returns (empty);
}

